packages/mdc-chips/chip-set/foundation.js

```diff
@@ -45,6 +53,42 @@ class MDCChipSetFoundation extends MDCFoundation {
    */
   constructor(adapter) {
     super(Object.assign(MDCChipSetFoundation.defaultAdapter, adapter));
+
+    /** 
+     * The active chips in the set. Only used for choice chip set or filter chip set.
+     * @private {!Array<!MDCChip>}
+     */
+    this.activeChips_ = [];
```

Comment:
I'm guessing this is an array because you'll need to support multiple selected chips in the future?

Reply:
Yep, specifically in the `--filter` variant.